bin/kernelstub

```diff
@@ -19,6 +18,9 @@ OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
 
+Please see the provided LICENCE.txt file for additional distribution/copyright
```

Comment:
LICENSE is misspelled

Reply:
In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.